src/command_modules/azure-cli-network/azure/cli/command_modules/network/generated.py

```diff
@@ -162,38 +159,35 @@
+cli_command('network express-route auth show', ExpressRouteCircuitAuthorizationsOperations.get, factory)
+cli_command('network express-route auth list', ExpressRouteCircuitAuthorizationsOperations.list, factory)
+cli_command('network express-route auth create', ExpressRouteCircuitAuthorizationsOperations.create_or_update, factory)
 
 # ExpressRouteCircuitPeeringsOperations
 factory = lambda _: _network_client_factory().express_route_circuit_peerings
-cli_command('network express-route circuit-peering delete', ExpressRouteCircuitPeeringsOperations.delete, factory)
-cli_command('network express-route circuit-peering show', ExpressRouteCircuitPeeringsOperations.get, factory)
-cli_command('network express-route circuit-peering list', ExpressRouteCircuitPeeringsOperations.list, factory)
-cli_generic_update_command('network express-route circuit-peering update', ExpressRouteCircuitPeeringsOperations.get, ExpressRouteCircuitPeeringsOperations.create_or_update, factory)
-
-factory = lambda _: get_mgmt_service_client(ExpressRoutePeeringClient).express_route_peering
-cli_command('network express-route circuit-peering create', ExpressRoutePeeringOperations.create_or_update, factory, transform=DeploymentOutputLongRunningOperation('Starting network express-route circuit-peering create'))
+cli_command('network express-route peering delete', ExpressRouteCircuitPeeringsOperations.delete, factory)
+cli_command('network express-route peering show', ExpressRouteCircuitPeeringsOperations.get, factory)
+cli_command('network express-route peering list', ExpressRouteCircuitPeeringsOperations.list, factory)
+cli_generic_update_command('network express-route peering update', ExpressRouteCircuitPeeringsOperations.get, ExpressRouteCircuitPeeringsOperations.create_or_update, factory, setter_arg_name='peering_parameters')
+cli_command('network express-route peering create', create_express_route_peering, factory)
 
 # ExpressRouteCircuitsOperations
 factory = lambda _: _network_client_factory().express_route_circuits
-cli_command('network express-route circuit delete', ExpressRouteCircuitsOperations.delete, factory)
-cli_command('network express-route circuit show', ExpressRouteCircuitsOperations.get, factory)
-cli_command('network express-route circuit get-stats', ExpressRouteCircuitsOperations.get_stats, factory)
-cli_command('network express-route circuit list-arp-tables', ExpressRouteCircuitsOperations.list_arp_table, factory)
-cli_command('network express-route circuit list-route-tables', ExpressRouteCircuitsOperations.list_routes_table, factory)
-cli_command('network express-route circuit list', list_express_route_circuits)
-cli_generic_update_command('network express-route circuit update', ExpressRouteCircuitsOperations.get, ExpressRouteCircuitsOperations.create_or_update, factory)
+cli_command('network express-route delete', ExpressRouteCircuitsOperations.delete, factory)
```

Comment:
Network was fine with the change. The logic is that an ExpressRoute _is_ a circuit, just like a KeyVault is a vault. Our KeyVault commands aren't `keyvault vault foo`. If an "ExpressRoute" could have many circuits then I would argue it makes sense, but the circuit is the fundamental _thing_ for an ExpressRoute so the circuit term is redundant.